WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -112,6 +113,9 @@ public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.main_activity);
 
+        // Allow vector drawables on pre-Lollipop (i.e. < API 21).
+        AppCompatDelegate.setCompatVectorFromResourcesEnabled(true);
```

Comment:
> If you update the configuration manually, then you probably do not want to enable this.

That's why I enabled it in `WPMainActivity.OnCreate` rather than `WordPress.OnCreate`.  We manually update the configuration using `onConfigurationChanged()` and `android:configChanges` in multiple activity and fragments throughout the app.

If you think it's better in `WordPress.OnCreate`, I can cherry pick your commit or just wait to merge with `develop`.